app/controllers/dashboards_controller.rb

```diff
@@ -13,10 +18,28 @@ def refresh_dashboard
 
   private
 
-  def dashboard
+  def active_dashboard
+    if dashboard.user_role == "solicitor"
+      @_active_dashboard ||= dashboard(:not_finished)
+    else
+      @_active_dashboard ||= dashboard
+    end
+  end
+
+  def closed_dashboard
+    @_closed_dashboard ||= dashboard(:finished, :aborted)
+  end
+
+  def set_dashboard_count
+    @active_count ||= active_dashboard.defence_requests.count
+    @closed_count ||= closed_dashboard.defence_requests.count
+  end
+
+  def dashboard(*additional_scopes)
```

Comment:
We need a way of scoping the dashboard DR's depending on when its being used. The policy scopes are the 'top level' scopes that a role can see. we need finer grain control than that.